ITRStartTask/StoriesView/Views/StoriesViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  StoriesViewController.swift
+//  ITRStartTask
+//
+//  Created by Homyakov, Ilya2 on 21/08/2018.
+//  Copyright © 2018 Homyakov, Ilya2. All rights reserved.
+//
+
+import UIKit
+
+class StoriesViewController: UIViewController {
+    var presenter: StoriesPresenterProtocol!
+
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var rootActivityIndicatorView: UIActivityIndicatorView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.tableView.delegate = self
+        self.tableView.dataSource = self
+        self.tableView.register(UINib(nibName: "StoryCell", bundle: nil), forCellReuseIdentifier: "storyCell")
```

Comment:
Cell nib name and reuse ID should be move to string constant.

Reply:
Fixed